build.gradle

```diff
@@ -18,6 +18,7 @@ plugins {
 
     id "pl.allegro.tech.build.axion-release" version "1.17.2"
 
+    id "com.palantir.docker" version "0.36.0"
```

Comment:
I don´t understand, why we should use gradle to build the container. 
This can be handled by a github action very easily and if you want to build locally... just use docker build... 
Also the plugin is under lifetime support. (Security and Dep updates only)
I´d suggest to just add a workflow like that one as soon we start building containers via pipeline:
https://docs.docker.com/build/ci/github-actions/

But leave as is until we start building containers. :)